src/dotnet-test-xunit/project.json

```diff
@@ -17,25 +17,30 @@
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
-        "dnxcore50",
+        "dotnet",
+        "portable-dnxcore50+net45+win8+wp8+wpa81",
         "portable-net45+win8"
       ]
     },
     "netstandardapp1.5": {
       "imports": [
-        "dnxcore50",
+        "dotnet",
+        "portable-dnxcore50+net45+win8+wp8+wpa81",
         "portable-net45+win8"
       ]
     }
   },
   "dependencies": {
-    "Microsoft.NETCore.App": "1.0.0-rc2-23931",
-    "Microsoft.Extensions.Testing.Abstractions": "1.0.0-beta-002254",
-    "Microsoft.Extensions.CommandLineUtils.Sources": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-24008",
+      "type": "platform"
+    },
+    "Microsoft.Extensions.Testing.Abstractions": "1.0.0-rc2-002353",
+    "Microsoft.Extensions.CommandLineUtils": {
```

Comment:
The .Sources package is no longer being maintained. Switched to the non-sources package which targets netstandard. Would be nice to change this to use `System.CommandlLine` since the parse behavior is between